StortSpelProjekt/Engine/src/Renderer/Renderer.cpp

```diff
@@ -628,7 +628,7 @@ void Renderer::updateMousePicker()
 	}
 }
 
-void Renderer::initRenderTasks()
+void Renderer::initRenderTasks(const Window* window)
```

Comment:
Isn't window saved as a member variable in Renderer?

Reply:
No?